android/app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -24,66 +26,48 @@
     private String name;
     private Double latitude;
     private Double longitude;
-    @JsonProperty("location-name")
     private String locationName;
-    @JsonProperty("starts-at")
-    private String startTime;
-    @JsonProperty("ends-at")
-    private String endTime;
+    private String startsAt;
```

Comment:
@iamareebjamal It is needed to rename property name for Supporting naming strategy.
Regarding migration, i think migration is needed when models changes and old app is installed. IMO as we are working on development branch and to make development process fast we should avoid migration for now. 
Also i think app generator uses master branch so we can add all migrations by comparing both branch when merging development to master.